laterpay/application/Controller/Frontend/Post.php

```diff
@@ -91,7 +91,7 @@ public function ajax_redeem_voucher_code( LaterPay_Core_Event $event ) {
             throw new LaterPay_Core_Exception_InvalidIncomingData( 'link' );
         }
 
-        // check, if voucher code exists and time pass is available for purchase
+        // check, if voucher code exists and time pass or subscription is available for purchase
```

Comment:
comments should end it full stop (.)

Reply:
This is an existing comment but if we are changing an existing we can update with wpcs.